### ISSUE TITLE
Compatibility with Numba > 0.48

### ISSUE THREAD

**Issue by evetion** [openstreams/wflow#35]
Using Numba greater than 0.48 will produce

```bash
  File "/opt/conda/envs/wflow/lib/python3.7/site-packages/wflow-2020.1.2.dev8+gfa963f4d.d20200901-py3.7.egg/wflow/wflow_sbm.py", line 75, in <module>
    from wflow.wflow_funcs import *
  File "/opt/conda/envs/wflow/lib/python3.7/site-packages/wflow-2020.1.2.dev8+gfa963f4d.d20200901-py3.7.egg/wflow/wflow_funcs.py", line 32, in <module>
    from numba.errors import NumbaPendingDeprecationWarning

ERROR:root:<class 'ModuleNotFoundError'>: No module named 'numba.errors'
```

Can be at least guarded against by #34. Possibly related to #29.

**Comment by LBuckma** [comment]
I have the same issue. Using numba<0.48 causes other compatibility issues. The only solution I can find is to change wflow_func.py line 32 from "from numba.errors import NumbaPendingDeprecationWarning" to "from numba.core.errors import NumbaPendingDeprecationWarning"

**Comment by visr** [comment]
Yes thanks for the report. I believe a7c5b8442722832915d8165d3afa2bc082c68c54 should help here, which is part of #34.

**Comment by visr** [comment]
Fixed in #34
